ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/CCodeGenerator.java

```diff
@@ -64,12 +63,12 @@ public CCodeGenerator() {
 
     @Override
     public void generate(final Ds3ApiSpec spec, final FileUtils fileUtils, final Path destDir) throws IOException {
-        this.spec = spec;
         this.fileUtils = fileUtils;
 
         try {
             final ImmutableList<Enum> allEnums = getAllEnums(spec);
-            final ImmutableList<Struct> allStructs = getStructsOrderedList(spec);
+            final Set<String> enumNames = allEnums.stream().map(Enum::getName).collect(Collectors.toSet());
```

Comment:
erm, no.  Last time it was to avoid .build()ing multiple times in a loop, but and ImmutableSet does work here.

Will change.